runtime/src/tasks/fifo/tasks-fifo.c

```diff
@@ -906,7 +906,7 @@ static void maybe_add_thread(void) {
                 "max threads per locale is unbounded"
                 ", but unable to create more than %d threads",
                 num_threads);
-      chpl_warning(msg, 0, 0);
+      chpl_task_warnNumThreadsPerLocale(msg);
```

Comment:
I don't think we want to use the wrapper here. I think the wrapper makes sense to quiet warnings we issues when the user has set `CHPL_RT_NUM_THREADS_PER_LOCALE`, but that's not the case here.

Reply:
Gotcha that makes sense